CODEOWNERS

```diff
@@ -1 +1,2 @@
 .github/* @fouge @TheButlah @vmenge @oldgalileo
+orb-slot-ctrl/* @vmenge @oldgalileo @alekseifedotov
```

Comment:
I don't want to add individual codeowners to specific software components. Anyone should be able to review code in this repo. We do not need to further bottleneck our review process.

Reply:
I'm okay with postponing this discussion for now, but I strongly feel a cross-team monorepo does not work well without a _CODEOWNERS_ file. IMO every crate / lib should have a team as clear owner as long as we're not bottlenecked by a single person.